pkl-core/src/test/kotlin/org/pkl/core/EvaluateTestsTest.kt

```diff
@@ -344,21 +344,21 @@ class EvaluateTestsTest {
     assertFalse(results.examples().hasError())
 
     val fail1 = res.failures[0]
-    assertThat(fail1.render().stripFileAndLines(tempDir))
+    assertThat(fail1.message.stripFileAndLines(tempDir))
       .isEqualTo(
         """
-      (/tempDir/example.pkl)
-      Expected: (/tempDir/example.pkl-expected.pcf)
-      new {
-        name = "Alice"
-        age = 45
-      }
-      Actual: (/tempDir/example.pkl-actual.pcf)
-      new {
-        name = "Bob"
-        age = 33
-      }
-    """
+        #0 (/tempDir/example.pkl):
+          Expected: (/tempDir/example.pkl-expected.pcf)
+          new {
+            name = "Alice"
+            age = 45
+          }
+          Actual: (/tempDir/example.pkl-actual.pcf)
+          new {
+            name = "Bob"
+            age = 33
+          }
```

Comment:
I think these need an extra space so that the `Expected` and `Actual` align with the `(` column

Reply:
I've discussed it more [here](https://github.com/apple/pkl/pull/738#discussion_r1819607141), TL;DR yes, it's a moving target but IMHO it could be solved to a great extent just by forcing alignment with double-digit examples.

The suggestion is aesthetic but I think it's quite reasonable and helpful UX-wise. Maybe it's just me but I do find misalignments (in this case, the breaking of the vertical whitespace) distracting when looking at code / logs 😅